Applications/JupyterHub.qmd

```diff
@@ -3,72 +3,47 @@ title: "JupyterLab"
 image: _images/jupyter_login.png
 ---
 
-JupyterLab is an advanced interactive development environment (IDE) that offers a flexible and feature-rich interface for working with notebooks, code, and data. It allows users to organize their workspaces using a flexible layout system with panels, views, and tabs for various activities. Furthermore, it supports various document formats, including notebooks, text files, code files, and markdown files. With its modular and extensible architecture, JupyterLab enables customization through extensions, additional functionalities, and integration with external tools. 
-It enhances the user experience with features like a file browser, command palette, debugger, and console, making it a versatile tool for interactive data exploration, analysis, and scientific computing.
+JupyterLab is a user-friendly tool for working with data and code. User can customize it by adding extra features and connecting it with other software. This makes organizing work, rectifying mistakes, and interacting with data easier. For more detailed information, please visit the [JupyterLab documentation](https://jupyterlab.readthedocs.io/en/stable/){target="_blank"}.
 
+Within the Copernicus Dataspace ecosystem, a free of charge jupyterkab service is offered. It allows users to access and analyze Earth observation data effectively. 
 
-You can access the JupyterLab service by clicking on this link.: [https://jupyterhub.dataspace.copernicus.eu/](https://jupyterhub.dataspace.copernicus.eu/){target="_blank"}
+## Notebooks
 
-![](_images/JupyterLab_welcome.png)
+Within JupyterLab, python notebooks can be used as an interactive programming environment for those who want to prototype their EO data processing quickly. Each notebook is set-up in order to access the EO Data repository. Additionally, example notebooks are available to help users get started with their analysis. 
```

Comment:
the notebooks are not really set up to access the eo data repository easily
in fact, access from these notebooks is the same as from your own laptop, because data is not mounted, nor is there easy s3 access or anything like that.
I already logged an issue about it, but it doesn't get solved for some reason, so perhaps we should adjust docs to avoid overselling things.